<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>My</class>
 <widget class="QWidget" name="My">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1441</width>
     <height>991</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#background {  
    border-image:url(:/icons/background.png) 0 0 0 0 stretch stretch;
	background-size: contain;  
}</string>
   </property>
   <widget class="QPushButton" name="goback">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt;</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt ;</string>
    </property>
    <property name="text">
     <string>请输入昵称</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>咖啡种类</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSet">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>设置当日数量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRead">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>读取数据</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>511</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>720</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>720</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>已经喝了</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>拿铁</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>卡布奇诺</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>摩卡</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>浓缩咖啡</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="leCount">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>咖啡数量</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>显示数据</string>
    </property>
   </widget>
   <widget class="QLabel" name="ImageLabel">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>230</y>
      <width>561</width>
      <height>542</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="AdviceLabel">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>140</y>
      <width>600</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>goback</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label</zorder>
   <zorder>btnSet</zorder>
   <zorder>btnRead</zorder>
   <zorder>plainTextEdit</zorder>
   <zorder>progressBar</zorder>
   <zorder>label_3</zorder>
   <zorder>comboBox</zorder>
   <zorder>leCount</zorder>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>AdviceLabel</zorder>
   <zorder>ImageLabel</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
